src/storage/datacore/restapi/mode/listpool.pm

```diff
@@ -0,0 +1,95 @@
+#
+# Copyright 2024 Centreon (http://www.centreon.com/)
+#
+# Centreon is a full-fledged industry-strength solution that meets
+# the needs in IT infrastructure and application monitoring for
+# service performance.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+package storage::datacore::restapi::mode::listpool;
+use strict;
+use warnings;
+
+use base qw(centreon::plugins::mode);
+
+sub new {
+    my ($class, %options) = @_;
+    my $self = $class->SUPER::new(package => __PACKAGE__, %options);
+    bless $self, $class;
+
```

Comment:
Maybe useful to ad a `filter-name`option that is frequent in service discovery modes. Something like this : 
```
    $options{options}->add_options(arguments => {
        'filter-name:s' => { name => 'filter_name' },
    });
    ```